src/Object/Post.php

```diff
@@ -799,8 +799,7 @@ private function getCommentBox($indent)
 				'$edquote'     => L10n::t('Quote'),
 				'$edcode'      => L10n::t('Code'),
 				'$edimg'       => L10n::t('Image'),
-				'$edurl'       => L10n::t('Link'),
-				'$edvideo'     => L10n::t('Video'),
+				'$edurl'       => L10n::t('Link or Media'),
```

Comment:
Please keep two distinct buttons:
- `Link`: BBCode `[url]` tag insertion with the chain icon
- `Attach`: Replacing the `Video` button with a paperclip icon.

Reply:
I guess I misunderstood you suggestion. Why do you want to keep the link button which inserts the `[url]` and an attach button which also insert the `[url]` tag but already filled with the url ?

Reply:
Because sometime I want highlight something I wrote, add a BBCode link tag and fill either the URL or the title myself.

The attach button on the other hand replaces the audio and video buttons, while keeping consistency with the new post formatting button.

Reply:
I see. What would you say if the prompt will inser the empty `[url]` tag if no link is entered in the prompt since it is the default ?

Reply:
I don't like the prompt at all, but I agree it should be consistent with the new post form. Besides, this would be confusing since we support highlighting text, clicking on the BBCode tag button and the generated BBCode will wrap the selected text.

Reply:
The other side of this PR is that I'd like a BBCode `[url]` button for the new post form since it was missing so far 😁